app/brave_main_delegate.cc

```diff
@@ -134,7 +134,6 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
   command_line.AppendSwitch(switches::kEnableDomDistiller);
 #endif
   command_line.AppendSwitch(switches::kDisableDomainReliability);
-  command_line.AppendSwitch(switches::kDisableChromeGoogleURLTrackingClient);
```

Comment:
what happened with this?

Reply:
Part of this commit: https://github.com/brave/brave-core/pull/3600/commits/75b7975c4694d39b848a89a362301d1fd4315d75

Due to https://chromium.googlesource.com/chromium/src/+/2e9897792488b3940ebb18b0071cfced7e90e3b1 (the link in the commit is wrong).